include_list.txt

```diff
@@ -0,0 +1,15 @@
+# This is a list of Python additional packages needed. This file is used with pip.
+
+# ----- IDF v4.3 and v4.4 -----
+# cffi 1.15 supports Python 3.6+ only
+cffi<1.15;python_version<'3.6'
+
+# esptool requirements (see components/esptool_py/esptool/setup.py)
+reedsolo>=1.5.3,<=1.5.4
+# bitstring 4 dropped support for Python 3.6 and older
+bitstring>=3.1.6,<4
+ecdsa>=0.16.0
+
+# espcoredump requirements
+# This is the last version supports both 2.7 and 3.4
```

Comment:
Also we don't build for these. `construct` is pure Python so I think we can remove the following restriction.